src/status_im/contexts/wallet/send/transaction_progress/view.cljs

```diff
@@ -27,9 +27,10 @@
 
 (defn- footer
   [{:keys [color leave-page]}]
-  (let [save-address-visible? false]
+  (let [send-to-address      (rf/sub [:wallet/wallet-send-to-address])
+        save-address-hidden? (rf/sub [:wallet/address-saved? send-to-address])]
     [quo/bottom-actions
-     {:actions          (if save-address-visible? :two-actions :one-action)
+     {:actions          (if-not save-address-hidden? :two-actions :one-action)
```

Comment:
I think there is a general preference to avoid if-not and just swap the order, particularly for simple cases like this.
`(if save-address-hidden?  :one-action :two-actions)`
I believe it makes it slightly less effort to process